### DISCUSSION TITLE
Any working testnet faucet?

### DISCUSSION THREAD

**Discussion by DanL0** [aptos-labs/aptos-developer-discussions#552] [Questions]
### Describe your question in detail.

I have tried three officially recommended ways of getting funds on testnet and none of them work. Usually it errors with 500 error.

I tried getting into Discord to ask this question but verification is broken there as well. Clicking "Get verified" results in "interaction failed".

I have to admit testnet onboarding experience in Aptos so far hasn't been great. Is there any working faucet?

### What error, if any, are you getting?

```
➜  aptos account fund-with-faucet --account XYZ
{
  "Error": "API error: Faucet issue: FaucetClientError { inner: Inner { kind: HttpStatus(500), source: None } }"
}
```

![image](https://github.com/user-attachments/assets/19a86267-a0e8-4618-9286-7be2e48e5cf7)

<img width="624" alt="image" src="https://github.com/user-attachments/assets/6395c529-ec02-4c88-93a6-ee8609a0d1ff">

**Comment by Levi0804** [comment]
Hey, I'm encountering the same error. You can try the same on devnet, or share your public address with me and I’ll send you some testnet tokens for the time being.

**Reply by DanL0** [reply]
@Levi0804 

Ideally if you could send some testnet tokens:

```
0x6070a006c8b26eb6054e5b78b9c906f63e53fc80ec6c4d8708cbb0577d5a6d4c
```

**Reply by Levi0804** [reply]
> @Levi0804
> 
> Ideally if you could send some testnet tokens:
> 
> ```
> 0x6070a006c8b26eb6054e5b78b9c906f63e53fc80ec6c4d8708cbb0577d5a6d4c
> ```

sent

**Reply by DanL0** [reply]
Thank you very much sir.

**Reply by Levi0804** [reply]
No problem :)

**Comment by shankars99** [comment]
encountering the same issue

0x3d24005f22a2913a9e228547177a01a817fcd5bbaa5290b07fe4826f3f31be4a

could I get some at this address?

**Reply by Levi0804** [reply]
done!

**Comment by RoDmitry** [comment]
It now requires a Google token to fund an account on `testnet`, but you can use https://aptos.dev/network/faucet
